### ISSUE TITLE
Can't set editor font

### ISSUE THREAD

**Issue by scenzhang** [Microsoft/vscode#50407]
Issue Type: <b>Bug</b>

- Change editor.fontFamily in settings.json 
- Restart VSCode
- Editor font remains the default

I thought it might have been an issue with the particular font I was trying to use (Hasklig), but changing it to just 'Courier New' or 'Arial' didn't work either. Tried with both extensions enabled and disabled.

VS Code version: Code 1.23.1 (d0182c3417d225529c6d5ad24b7572815d0de9ac, 2018-05-10T16:03:31.083Z)
OS version: Darwin x64 17.5.0

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz (8 x 2900)|
|GPU Status|2d_canvas: enabled<br>flash_3d: enabled<br>flash_stage3d: enabled<br>flash_stage3d_baseline: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>native_gpu_memory_buffers: enabled<br>rasterization: enabled<br>video_decode: enabled<br>video_encode: enabled<br>vpx_decode: enabled<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|2, 5, 5|
|Memory (System)|16.00GB (1.04GB free)|
|Process Argv|/Applications/Visual Studio Code.app/Contents/MacOS/Electron|
|Screen Reader|no|
|VM|0%|

</details><details><summary>Extensions (54)</summary>

Extension|Author (truncated)|Version
---|---|---
All-Autocomplete|Ati|0.0.12
bracket-pair-colorizer|Coe|1.0.55
gc-excelviewer|Gra|2.0.21
vscode-docker|Pet|0.0.27
code-settings-sync|Sha|2.9.2
quokka-vscode|Wal|1.0.125
better-comments|aar|1.2.3
Bookmarks|ale|0.19.1
react-pure-to-class-vscode|ang|1.1.4
snapshot-tools|asv|0.0.7
django-html|bib|1.2.0
unique-lines|bib|1.0.0
htmltagwrap|bra|0.0.7
react-food-truck|bur|1.0.2
simple-react-snippets|bur|1.1.1
vscode-exports-autocomplete|cap|0.5.4
solargraph|cas|0.17.3
npm-intellisense|chr|1.3.0
very-smart-select|clo|0.3.0
vscode-eslint|dba|1.4.10
vscode-quick-select|dba|0.2.5
gitlens|eam|8.3.2
vscode-html-css|ecm|0.2.0
vscode-npm-script|eg2|0.3.4
prettier-vscode|esb|1.3.1
auto-close-tag|for|0.5.6
auto-rename-tag|for|0.0.15
macros|ged|1.2.1
vscode-atom-javascript-snippet|glu|0.2.1
css2react|got|0.0.6
react-native-snippet|jun|0.4.3
vscode-colorize|kam|0.6.18
theme-karyfoundation-themes|kar|18.0.1
vscode-gist|ken|1.2.0
code-beautifier|mic|2.0.3
dotenv|mik|1.0.1
python|ms-|2018.4.0
cpptools|ms-|0.17.3
Go|ms-|0.6.80
debugger-for-chrome|msj|4.4.3
indent-rainbow|ode|7.2.2
vscode-subword-navigation|ow|1.2.0
bracketeer|pus|1.1.0
Ruby|reb|0.18.0
vscode-icons|rob|7.23.0
gi|rub|0.2.11
partial-diff|ryu|1.1.0
vscode-djaneiro|the|1.4.1
es-quotes|vil|0.2.6
expand-selection-to-scope|vit|0.2.0
vscode-react-native|vsm|0.6.10
jinja|who|0.0.8
change-case|wma|1.0.0
JavaScriptSnippets|xab|1.6.0

(2 theme extensions excluded)

</details>

**Comment by weinand** [comment]
@scenzhang please provide detailed steps how you changed the font.

**Comment by scenzhang** [comment]
@weinand I opened settings.json and copied the line `"editor.fontFamily": "Menlo, Monaco, 'Courier New', monospace"` from the defaults into the user settings. I then added 'Hasklig' to the comma separated list, so it now reads `"editor.fontFamily": "Hasklig, Menlo, Monaco, 'Courier New', monospace"`. I also tried configurations like `"'Courier New', monospace"` or `"Arial"` but neither of those changed anything.
